stdlib/public/core/CMakeLists.txt

```diff
@@ -121,7 +120,6 @@ set(SWIFTLIB_SOURCES
   ### PLEASE KEEP THIS LIST IN ALPHABETICAL ORDER ###
   Availability.swift
   Bit.swift
-  CollectionMirrors.swift.gyb
   CollectionOfOne.swift
   ExistentialCollection.swift.gyb
   Mirror.swift
```

Comment:
Speaking of Mirror.swift, the `_legacyMirror` method should obviously be gone when this PR is done.